includes/admin/class-wcs-admin-post-types.php

```diff
@@ -684,43 +684,43 @@ public function render_shop_subscription_columns( $column, $subscription = null
+		$date_type      = array_key_exists( $column, $date_type_map ) ? $date_type_map[ $column ] : $column;
+		$date_timestamp = $subscription->get_time( $date_type );
+
+		if ( 0 === $date_timestamp ) {
+			return '-';
+		}
+
+		$datetime         = wcs_get_datetime_from( $date_timestamp );
+		$accurate_date    = $datetime->date_i18n( __( 'Y/m/d g:i:s A', 'woocommerce-subscriptions' ) );
+		$fuzzy_human_date = $subscription->format_date_to_display( $date_timestamp, $date_type );
+		$column_content   = sprintf(
+			'<time class="%s" title="%s">%s</time>',
+			esc_attr( $column ),
+			esc_attr( $accurate_date ),
+			esc_html( $fuzzy_human_date )
+		);
 
-		$date_type_map = array( 'last_payment_date' => 'last_order_date_created' );
-		$date_type     = array_key_exists( $column, $date_type_map ) ? $date_type_map[ $column ] : $column;
-		$datetime      = wcs_get_datetime_from( $subscription->get_time( $date_type ) );
-
-		if ( 0 == $subscription->get_time( $date_type, 'gmt' ) ) { // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
-			$column_content = '-';
-		} else {
-			$accurate_date    = $datetime->date_i18n( __( 'Y/m/d g:i:s A', 'woocommerce-subscriptions' ) );
-			$fuzzy_human_date = $subscription->get_date_to_display( $date_type );
-			$column_content   = sprintf(
-				'<time class="%s" title="%s">%s</time>',
-				esc_attr( $column ),
-				esc_attr( $accurate_date ),
-				esc_html( $fuzzy_human_date )
-			);
-
-			// Custom handling for `Next payment` date column.
-			if ( 'next_payment_date' === $column ) {
-				$subscription_is_active = $subscription->has_status( 'active' );
+		// Custom handling for `Next payment` date column.
+		if ( 'next_payment_date' === $column ) {
+			$subscription_is_active = $subscription->has_status( 'active' );
 
-				$tooltip_message = '';
-				$tooltip_classes = 'woocommerce-help-tip';
+			$tooltip_message = '';
+			$tooltip_classes = 'woocommerce-help-tip';
 
-				if ( $subscription_is_active && $datetime->getTimestamp() < time() ) {
-					$tooltip_message .= __( '<b>Subscription payment overdue.</b></br>', 'woocommerce-subscriptions' );
-					$tooltip_classes .= ' wcs-payment-overdue';
-				}
+			if ( $subscription_is_active && $datetime->getTimestamp() < time() ) {
+				$tooltip_message .= __( '<b>Subscription payment overdue.</b></br>', 'woocommerce-subscriptions' );
+				$tooltip_classes .= ' wcs-payment-overdue';
+			}
 
-				if ( $subscription->payment_method_supports( 'gateway_scheduled_payments' ) && ! $subscription->is_manual() && $subscription_is_active ) {
-					$tooltip_message .= __( 'This date should be treated as an estimate only. The payment gateway for this subscription controls when payments are processed.</br>', 'woocommerce-subscriptions' );
-					$tooltip_classes .= ' wcs-offsite-renewal';
-				}
+			if ( $subscription->payment_method_supports( 'gateway_scheduled_payments' ) && ! $subscription->is_manual() && $subscription_is_active ) {
```

Comment:
Should we move the `$subscription_is_active` check first in this condition since it's already an established variable and may avoid us instantiating payment gateways and running gateway support checks.

Reply:
Yeah good idea. While fixing this I actually noticed that both conditions start with `$subscription_is_active` can they could be consolidated. Then I realised that that the `$tooltip_message` only changes if the subscription is active and so the "active" check can be made a top level condition. 

I've fixed that in [759e7e9](https://github.com/Automattic/woocommerce-subscriptions-core/pull/789/commits/759e7e9b00e069ffdd9ae109cd5fc7002209a325)